examples/impute/plot_missing_values.py

```diff
@@ -107,17 +120,18 @@ def get_results(dataset):
 x_labels = ['Full data',
             'Zero imputation',
             'Mean Imputation',
-            'Multivariate Imputation']
-colors = ['r', 'g', 'b', 'orange']
+            'KNN Imputation',
+            'Iterative Imputation']
+colors = ['r', 'g', 'b', 'orange', 'black']
```

Comment:
If we change this example, I would be in favour to use the default colour from matplotlib.
However, we could edit the line and marker style.